app/src/main/java/com/pajato/android/gamechat/exp/model/Chess.java

```diff
@@ -11,23 +12,20 @@
 
 import static com.pajato.android.gamechat.exp.ExpType.chess;
 
-/**
- * Created by sscott on 12/30/16.
- */
-
 /** Provide a Firebase model class for a chess game experience. */
 @IgnoreExtraProperties
 public class Chess implements Experience {
 
     public final static int ACTIVE = 0;
-    public final static int RED_WINS = 1;
-    public final static int BLACK_WINS = 2;
+    public final static int PRIMARY_WINS = 1;
```

Comment:
An alternative to RED/BLACK or BLACK/WHITE, that is used in Wikipedia, is to use LIGHT/DARK.  This works for Chess, Checkers, and Go.